### PR TITLE
add-appxpackage: add French translation

### PR DESCRIPTION
**Pull request by Laykon4**
- [ ] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [ ] The page description(s) have links to documentation or a homepage.
- [ ] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.fr/windows/add-appxpackage.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Add-AppxPackage
+
+> Un utilitaire PowerShell pour ajouter un paquet d'applications signé (`.appx`, `.msix`, `.appxbundle`, `.appxbundle` et `.msixbundle`) à un compte utilisateur.
+> Plus d'information : <https://learn.microsoft.com/powershell/module/appx/Add-AppxPackage>.
+
+- Ajouter un paquet d'application :
```
Comment (noraj) [inline pages.fr/windows/add-appxpackage.md:6]:
```suggestion
- Ajoute un paquet d'application :
```

File: pages.fr/windows/add-appxpackage.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+`Add-AppxPackage -Path {{chemin\vers\paquet.msix}}`
+
+- Ajouter un paquet d'application avec ses dependences:
```
Comment (noraj) [inline pages.fr/windows/add-appxpackage.md:10]:
```suggestion
- Ajoute un paquet d'application avec ses dependences :
```

File: pages.fr/windows/add-appxpackage.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+`Add-AppxPackage -Path {{chemin\vers\paquet.msix}} -DependencyPath {{chemin\vers\dependences.msix}}`
+
+- Installer une application en utilisant le fichier d'installation de l'application:
```
Comment (noraj) [inline pages.fr/windows/add-appxpackage.md:14]:
```suggestion
- Installe une application en utilisant le fichier d'installation de l'application :
```

File: pages.fr/windows/add-appxpackage.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+`Add-AppxPackage -AppInstallerFile {{chemin\vers\application.appinstaller}}`
+
+- Ajouter un paquet non signé:
```
Comment (noraj) [inline pages.fr/windows/add-appxpackage.md:18]:
```suggestion
- Ajoute un paquet non signé :
```

**Review by noraj — CHANGES_REQUESTED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**